### ISSUE TITLE
[Lcd4Linux] plugin, ultimo4k, duo4kSe and probably others slow down zapping! 

### ISSUE THREAD

**Issue by nunigaia** [oe-alliance/oe-alliance-plugins#683]
Hi,

I just notice since severall days ago, that Vuplus STb´s running OpenVIX 6.3 and BlackHole 5.2 ( updated version ) slows down and it hangs after some time. I took this conclusion by done severall tests. 
As it devolves the spinner since I press any button on the remote control and try to navigate along the menus. it seems has to be anything related with LCD4Linux as I have also debugged the log:

Error: 

"File "/usr/lib/enigma2/python/Plugins/Extensions/LCD4linux/plugin.py", line 4689, in getPage
link = ensure_binary(link.encode('ascii', 'xmlcharrefreplace').decode().replace(' ', '%20').replace('\n', ''))
builtins.AttributeError: 'boundFunction' object has no attribute 'encode' "

Note: There´s no issues, if LCD4Linux is not installed.

regards
nunigaia

**Comment by MrServo** [comment]
Thanks for reporting! We have found the mistake and corrected it.

**Comment by jbleyel** [comment]
https://github.com/oe-alliance/oe-alliance-plugins/commit/4a001f5d6a38a208e87162f1ea091fc47858a942

**Comment by nunigaia** [comment]
No problem about reporting the issue. 
More you are free to ask me, for any test you want in order to fix it, if needed!

>
Thanks for reporting! We have found the mistake and corrected it.

> 

Not sure if this enough as I got now another error:

File "/usr/lib/enigma2/python/Plugins/Extensions/LCD4linux/plugin.py", line 4688, in getPage
if "User-Agent" not in headers:
builtins.TypeError: argument of type 'NoneType' is not iterable
